liota-lite/requirements.txt

```diff
@@ -1,7 +1,5 @@
 aenum==1.4.5
 linux-metrics==0.1.4
-mock==2.0.0
-paho-mqtt==1.2
+paho-mqtt==1.3.1
```

Comment:
I think there is something really wrong with the way liota-lite is designed. There should be a way to have a common file structure to describe the requirements and setup.py, with a flag being passed. This is a developer's nightmare to keep files in sync for liota and liota-lite.

Reply:
You can raise a separate bug to fix this later, as it is not a concern related to this CL.

Reply:
I don't believe this is required to be fixed as this approach is given to us by Open Source team.